service/src/main/java/uk/gov/hmcts/reform/fpl/config/HmctsCourtLookupConfiguration.java

```diff
@@ -33,10 +34,12 @@ public Court getCourt(String localAuthorityCode) {
     public static class Court {
```

Comment:
@akomisarek could we start to use Lombok here and remove constructor, getters?

Reply:
Done 👍